space_age/spec/features/user_can_delete_account_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
+
+RSpec.feature "registered user can delete account" do
+  include FeatureHelper
+  context "with valid params" do
+    scenario "they see the welcome page" do
+      @user = create(:user)
+
+      user_login
+      expect(current_path).to eq(packages_path)
+
+      visit '/dashboard'
+
+      expect(page).to have_content @user.username
+      expect(page).to have_content @user.email
+
+      click_on "Delete My Account"
+
+      expect(current_path).to eq(root_path)
+      expect(page).to_not have_content @user.username
+      expect(page).not_to have_content @user.email
+      expect(page).to have_content "Account Successfully Deleted"
+      # within("#title-bar-left") do
+      #   expect(page).to have_content "Login"
+      #   expect(page).to_not have_content "Logout"
+      # end
+    end
```

Comment:
Yeah I left these on purpose because I thought it would be a good test for knowing if you're still logged in and knew I would forget if I deleted it.